<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sleep Timer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>251</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Timer</string>
    </property>
   </widget>
   <widget class="QDial" name="dialMins">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnTime">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Shutdown Mode</string>
    </property>
    <widget class="QRadioButton" name="radShutdown">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>109</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Shutdown</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radRestart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>109</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Restart</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radSleep">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>109</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Sleep</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radHibernate">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>109</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Hibernate</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnUp">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDown">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>140</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Minutes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>265</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="statusTip">
    <string extracomment="Help"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
